src/main/java/no/finn/unleash/util/UnleashConfig.java

```diff
@@ -26,6 +26,7 @@
     private final long sendMetricsInterval;
     private final boolean disableMetrics;
     private final UnleashContextProvider contextProvider;
+    private final boolean syncFirstFetch;
```

Comment:
I don't fancy this name for this option. Suggestions welcomed!

Reply:
Thanks I like `syncFetchOnInit`!

Reply:
is `sync` for `synchronous` or `synchronizing`? Not a big fan of abbreviations in code, we all have editors that autocompletes for us anyways 🙂

Reply:
So `synchronousFetchOnInitialization` then?